docs/_docs/Instructors/04-Recommendations-High-Quality-materials.md

```diff
@@ -1,64 +1,62 @@
+The topic you're trying to explain might now be too second nature to you. Invest time into thinking about how to explain it intuitively. This step is the more critical the better you know the subject. 
 
-Googling the keywords _“ intuitions on ____ “_ and reading a couple of articles to see how other people explain it is a good way to get warmed up. 
+Googling the keywords _“ intuitions on ____ “_ and reading a couple of articles to see how other people explain it is a good way to start. 
 
-You will often find that the best explanations online are superb not because they are highly technical, but because they transmit the concepts in a **highly intuitive manner**. You can, and should, copy some of these explanations, naturally giving attribution to the author. Searching for intuitive explanations will also probably provide you with excellent pictures and ways of representing the problem that you would otherwise take a long time to build from scratch. 
+You will often find that the best explanations online are great not because they are highly technical, but because they transmit the concepts in a **highly intuitive manner**. You can, and should, copy some of these explanations, naturally giving credit to the author. Searching for intuitive explanations might also provide you with excellent pictures and ways of representing the problem that would otherwise take a long time to build from scratch. 
 
 ## Intuitive is more important than Precise 
-We are trying to create intuitions that the students can take in and remember. This will sometimes require sacrificing temporarity precision for intuition. Examples of this are: 
-> Saying "Correlation", when to be precise it should be "Pearson Correlation"   
+We are trying to create intuitions that the students can take in and remember. This will sometimes require sacrificing precision for intuition. Examples of this are: 
+* Saying "Correlation", when to be precise it should be "Pearson Correlation".
+* Saying that "generally using a Random Forest is a good place to start", instead of trying to qualify that _"except when the calibration of the tails of the probability distribution are more important than the rank ... [more advanced stuff which gets lost on the beginner student]"_ 
 
-> Saying that "generally using a Random Forest is a good place to start", instead of trying to qualify that _"except when the calibration of the tails of the probability distribution are more important than the rank ... [more impressive highly technical stuff which adds nothing to the student learning in the short term]"_ 
+You can alsways come back later with a simple _"remember when I said correlation? Well, technically I should have said Pearson correlation, because here's a new concept called Spearman"_ when it becomes relevant.
```

Comment:
```suggestion
You can always come back later with a simple _"remember when I said correlation? Well, technically I should have said Pearson correlation, because here's a new concept called Spearman"_ when it becomes relevant.
```